90-apex.hwdb

```diff
@@ -3,7 +3,7 @@
 # 3) sudo udevadm control --reload
 
 #keyboard:usb:v1038p120[02]*
-evdev:input:b0003v1038p120[0268]*
+evdev:input:b0003v1038p120[0268A]*
```

Comment:
Please note that on this line, there is missing `1600` for keyboard Apex M800. Is this intended? I needed to put my product ID here in order to be able to redefine default keys (and without correct code, the whole file is pointless).
Maybe there should be another PR for it.